src/UploadImage.tsx

```diff
@@ -228,9 +223,18 @@ export class UploadImage extends Component<Props> {
           type="file"
           style={{ display: "none", textAlign: "center" }}
           onChange={(e) => {
+            const argsPromises: Promise<CallbackArgs>[] = [];
             for (let i = 0; i < e.target.files.length; i += 1) {
-              this.uploadImage(e.target.files[i]);
+              argsPromises.push(this.uploadImage(e.target.files[i]));
             }
+            Promise.all(argsPromises)
+              .then((callbackArgsArray: CallbackArgs[]) =>
```

Comment:
I think you could use `Promise.allSettled` here, instead of `Promise.all`, given that each file upload is independent from all the others.

Reply:
Done